index.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Contributing to Open Source Part I: The Easy Way</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <div id="wrapper">
+        <nav>
+            <div class="container">
+                <div class="nav-logo">
+                    <img src="images/youtubelogo.png" alt="YouTube">
+                </div>
+                <div class="nav-searchbox">
+                    <input type="text" class="searchbox-input" placeholder="Enter">
+                    <button type="submit" class="searchbox-submit"> <img src="images/search.png" alt="search"> </button>
+                </div>
+                <div class="nav-socials">
+                    <a href="#" class="socials-button"><img src="images/globe.png" alt=""></a>
+                    <a href="#" class="socials-button"><img src="images/notification.png" alt=""></a>
+                    <a href="#" class="socials-button"><img src="images/avatar.png" alt=""></a>
+                </div>
+            </div>
+        </nav>
+        <div class="main-container">
+            <main class="primary-video">
+                <div>
+                    <iframe width="1179" height="650" src="https://www.youtube.com/embed/V74l_zS1x8E" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Please consider reducing the width of your main video to something like 850 and the height to 450. This will prevent the overlap with side videos.